configure.ac

```diff
@@ -11,6 +11,8 @@ m4_pattern_forbid([^AX_])
 
 AC_PROG_CC
 
+AC_DEFINE(_BSD_SOURCE, 1, [Required for struct timezone])
```

Comment:
Thanks. As configure is not the only way to compile Tig, I'd rather have the define done in include/tig/tig.h:37.

Reply:
updated, right below the config.h include

Reply:
Thanks. Looks like we also need to add an extra `#define _DEFAULT_SOURCE` for glibc >= 2.19.

Reply:
ah, right